index.html

```diff
@@ -265,7 +265,7 @@ <h2>Let's Talk</h2>
     <!-- Bootstrap Core JavaScript -->
     <script src="js/bootstrap.min.js"></script>
 
-    <script>(function(){var js,fs,d=document,id="tars-widget-script",b="https://tars-file-upload.s3.amazonaws.com/share/";if(!d.getElementById(id)){js=d.createElement("script");js.id=id;js.type="text/javascript";js.src=b+"js/widget.js";fs=d.getElementsByTagName("script")[0];fs.parentNode.insertBefore(js,fs)}})();window.tarsSettings = {"convid":"ByT8yG"};</script>
+    
```

Comment:
Thank goodness, bots are silly. This variable name is confusing. I think you have a bug here.